src/style.css

```
@@ -1,3 +1,96 @@
+
+
+
+:root {
+  --color-element: #00ACDC;
```

Use a more descriptive name for classes
```suggestion
  --main-color: #00ACDC;
```